src/components/MagicCodeModal.js

```diff
@@ -0,0 +1,94 @@
+import React, {PureComponent} from 'react';
```

Comment:
Did we conclude that  we should keep with the `MagicCode` motif in code vs `ValidateCodeModal` 
cc @NikkiWines

Reply:
Is this because ValidateCode page already exists in a different context?

Reply:
Good catch! I remember that a previous review @NikkiWines asked me to use validateCode in the code in another file. So I think it makes sense to change it here as well to be consistent.

Reply:
fixed.